frontend/src/v5/ui/controls/breadcrumbs/breadcrumbs.styles.ts

```diff
@@ -30,6 +30,7 @@ export const Container = styled(Breadcrumbs)`
 			overflow: hidden;
 			display: block;
 			white-space: nowrap;
+			height: 31px;
```

Comment:
This fixes all the segments that are white, but no the highlighted one (the rightmost one) which still is a bit uplifted
![image](https://user-images.githubusercontent.com/43547703/235645956-64d436e9-d7a2-476f-8e12-daa14ecd80cb.png)